packages/web/src/components/Error/ErrorPopup.tsx

```diff
@@ -146,6 +159,79 @@ export function AxiosErrorFailed({ url, status, statusText }: { url: string; sta
   )
 }
 
+export function BadAllocErrorMessage() {
+  const { t } = useTranslation()
+  return (
+    <ErrorContainer>
+      <h5>{t('An error has occurred: out of memory (std::bad_alloc).')}</h5>
+
+      <section className="mt-3">
+        <Alert color="danger" fade={false}>
+          {t('Nextclade tried to allocate system memory, but no additional memory could be allocated.')}
+        </Alert>
+      </section>
+
+      <section className="mt-3">
+        <div>
+          {t(
+            'Nextclade algorithm runs on this computer (and not on remote server or in the cloud) and may require large amounts of computational resources to perform calculations. Please make sure that there is enough system memory available for Nextclade to operate.',
+          )}
+          <sup>1</sup>
+        </div>
+      </section>
+
+      <section className="mt-3">
+        {t('Possible solutions:')}
+        <Ul>
+          <Li>{t('Close unused web browser tabs, other applications and documents, to free up some memory.')}</Li>
+          <Li>
+            {t('Reduce number of processing threads in Nextclade\'s "Settings" dialog.')}
+            <sup>2</sup>
+          </Li>
+          <Li>
+            {t('Reduce the amount of analyzed data. For example, split large .fasta file to multiple smaller ones.')}
+          </Li>
+          <Li>
+            {t('Consider updating your web browser. Nextclade runs best on the latest versions of {{ browserList }}')}
```

Comment:
What about suggesting `Edge`? If I remember correctly didn't both bug reporters mention that with `Edge` the bug didn't occur?

Reply:
I just mention **latest versions** of some browsers with known good WebAssembly implementations, that we test periodically.

We don't monitor Edge routinely and it has low market share, so I would not give it any official mention in the app.

The fact that it worker in Edge might have also been a coincidence. For example, user might have closed Chrome with many tabs, freeing memory and that's why Edge worked. But we won't know for sure.